cfpb/consumerfinance.gov
# Simple chart tilemap prototype

Closes https://github.local/CFPB/super-regular/issues/280

Testing:
 - Pull branch
 - Build new chart with chart type "Tilemap"
 - Set data source to https://files.consumerfinance.gov/data/ccpi/index_state.csv

Should behave like a tilemap (labels, tooltips on hover, etc.) & you should be able to change the date with the select element above the map.
